server/channels/db/migrations/postgres/000129_scheduled_posts_indexes.up.sql

```diff
@@ -0,0 +1,3 @@
+CREATE INDEX IF NOT EXISTS idx_scheduledposts_userid_channel_id_scheduled_at ON ScheduledPosts (UserId, ChannelId, ScheduledAt DESC);
+CREATE INDEX IF NOT EXISTS idx_scheduledposts_scheduledat_id_id ON ScheduledPosts (ScheduledAt desc, Id);
```

Comment:
The ordering of the columns seems inconsistent. Why is `ScheduledAt` before `Id`? Also, is `Id`  the primary key? Should it not be created as part of the table creation? It is a bit difficult to review index additions without the original schema of the table.

The column order in a multi-column index is important in how the DB will use the index for queries that use a sub-set of the columns.

Reply:
Yeah, Id is a primary key, so I can remove the second and third indexes.